README.md

```diff
@@ -1,35 +1,20 @@
-# OSM_DataManipulation
+#Sateltiles
```

Comment:
look at this url to see how to write good readme file https://gist.github.com/jxson/1784669